composer.lock

```diff
@@ -1,6484 +0,0 @@
-{
```

Comment:
Why was this deleted?

Reply:
Don't need to lock dependencies across environments, as all dependencies are used for testing only. Therefore composer should fetch the most up-to-date dependencies to run testing via GitHub Actions and/or locally.